### PR TITLE
Tc issue 122

### PR DESCRIPTION
**Pull request by vsperez**
I haven't test it jet.

### CODE REVIEW

File: transitclockApi/src/main/java/org/transitclock/api/rootResources/TransitimeApi.java [hunk 1]
```diff
@@ -306,7 +306,12 @@ public Response getVehiclesDetails(@BeanParam StandardParameters stdParameters,
 			@QueryParam(value = "s") String stopId,
 			@Parameter(description=" For when determining which vehicles are generating the"
 					+ "predictions so can label minor vehicles",required=false)@QueryParam(value = "numPreds") 
-			@DefaultValue("3") int numberPredictions) throws WebApplicationException {
+			@DefaultValue("3") int numberPredictions
+			,
+			@Parameter(description=" Return only assigned vehicles",required=false)@QueryParam(value = "assgined")
```
Comment (nselikoff) [inline transitclockApi/src/main/java/org/transitclock/api/rootResources/TransitimeApi.java]: @vsperez small typo here: "assgined"

**Comment by nselikoff** [comment]
Related issue: https://github.com/TheTransitClock/transitime/issues/122

**Comment by scrudden** [comment]
@vsperez Have you run any tests on this change? By that I mean have you set up an instance with this change and checked that all the maps work as expected?

**Comment by vsperez** [comment]
Sean,

I haven’t tested the last change. I don’t recall  exactly what test I did, so I will do it again and I will let you know.

Best Regards
Vicente

**Comment by vsperez** [comment]
Hi Sean,

I was testing the branch, and I could make it work property, but I had some issues, that I am quite sure were solved. I think I am getting crazy!!!! (tc_issue_131 merged on march). I don’t see those changes in develop.

1.- TransitimeApi — getRouteDetails  missing the r parameter.  
2.- IpcRoute  — createStops using if( tripPattern.getDirectionId().compareTo(currentDirectionId)==0) instead of if( tripPattern.getDirectionId()==null || tripPattern.getDirectionId().compareTo(currentDirectionId)==0) (line 225 and 236)
3.- Some improvements in Synoptic view. By the way, with this agency I found some bugs.

I am testing with Intercity…But no buses right now :(. So it is working both: Map and Synoptic but without buses so far.


Regards.
Vicente

**Comment by scrudden** [comment]
Deprecated by pull request #142

**Pull request closed without merging**